snakemake/config.json

```diff
@@ -1,9 +1,43 @@
 {
-  "output_dir": "data",
-
+  "output_dir": "../data",
```

Comment:
We should probably keep this as `./data` (make the workflow not modify anything thats outside this folder)